warehouse/ingest-core/src/main/java/datawave/util/flag/FlagMaker.java

```diff
@@ -266,13 +266,19 @@ void loadFilesForDistributor(FlagDataTypeConfig fc, FileSystem fs) throws IOExce
                 } else {
                     try {
                         this.fd.addInputFile(new InputFile(folder, status, this.fmc.getBaseHDFSDir(), this.fmc.isUseFolderTimestamp()));
+                        logFileInfo(fc, status);
                     } catch (UnusableFileException e) {
                         log.warn("Skipping unusable file " + status.getPath(), e);
                     }
                 }
             }
         }
     }
+
+    protected void logFileInfo(FlagDataTypeConfig fc, FileStatus status) {
+        log.info("File type is " + fc.getDataName());
```

Comment:
Lets combine these lines into one to avoid too much log bloat:
log.info("File " + fc.getDataName() + " : " + status);

Reply:
sounds good changes made in update